src/spdx3/bump_from_spdx2/actor.py

```diff
@@ -29,9 +30,9 @@ def bump_actor(spdx2_actor: Spdx2_Actor, payload: Payload, creation_info: Creati
     name_without_whitespace = ''.join(name.split())
     if email:
         external_identifiers.append(ExternalIdentifier(ExternalIdentifierType.EMAIL, email))
-        spdx_id: str = f"SPDXRef-Actor-{name_without_whitespace}-{email}"
+        spdx_id: str = "#".join([document_namespace, f"SPDXRef-Actor-{name_without_whitespace}-{email}"])
```

Comment:
```suggestion
        spdx_id: str = f"{document_namespace}#SPDXRef-Actor-{name_without_whitespace}-{email}")
```
I think one f-string is more straight forward than mixing a join operation and a f-string. This might be consistent with the other changes where there is no f-string, but I would prefer to always use a f-string. Probably a matter of taste.